core/src/main/java/di/StoreModule.kt

```diff
@@ -45,4 +46,9 @@ class StoreModule {
     @Singleton
     @Provides
     fun provideActorsStore(): CreditsStore = CreditsStore()
+
+
+    @Singleton
+    @Provides
+    fun providePPeopleStore(): PPeopleStore = PPeopleStore()
```

Comment:
why do you need to provide this if you already create the class with inject const.?